spec/javascripts/controllers/containers/container_topology_controller_spec.js

```diff
@@ -2,7 +2,6 @@ describe('containerTopologyController', function() {
     var scope, $controller, $httpBackend;
     var mock_data =  getJSONFixture('container_topology_response.json');
     var replicator = { id:"396086e5-7b0d-11e5-8286-18037327aaeb",  item:{display_kind:"Replicator", name:"replicator1", kind:"ContainerReplicator", id:"396086e5-7b0d-11e5-8286-18037327aaeb", miq_id:"10"}};
-    var atomic_ent_provider = { id:"4",  item:{display_kind:"AtomicEnterprise", name:"myProvider", kind:"ContainerManager", id:"4", miq_id:"4"}};
```

Comment:
This feels wrong to have ids hardcoded in tests, but is not a question for this PR...more for @martinpovolny and/or @himdel